core/blockchain.go

```diff
@@ -29,6 +29,7 @@ import (
 	"sync/atomic"
 	"time"
 
+    "github.com/google/uuid"
```

Comment:
extra space here?

Reply:
Problem with my editor setting (spaces vs. tabs), should be fixed now